manager/controllers/advisories.go

```diff
@@ -18,13 +18,14 @@ var AdvisoriesOpts = ListOpts{
 }
 
 type AdvisoriesDBLookup struct {
-	ID string `query:"am.name"`
+	ID string `query:"am.name" gorm:"column:id"`
```

Comment:
Is it required to explicitly add "gorm" annotation even if the field is the same as column name?

Reply:
Yes, I dont know why, but without this, GORM will rename column to e.g. `publicdate` instead of `public_date`.